mlbstatsapi/mlbapi.py

```diff
@@ -75,7 +75,7 @@ def get_people_id(self, fullName) -> List[int]:
         # if mlbdata is not empty, and 'people' key is in mlbdata.data and mlbdata.data['people'] is not empty list
         if ('people' in mlbdata.data and mlbdata.data['people']): 
             for person in mlbdata.data['people']:
-                if person['fullName'] == fullName: # Match person fullName
+                if person['fullname'] == fullName: # Match person fullName
```

Comment:
A good idea might be to change this to:
`if person['fullname'].lower() == fullName.lower(): # Match person fullName`

Reply:
changed